shared/src/simulation/activities/transfer-patient-to-hospital.ts

```diff
@@ -0,0 +1,115 @@
+    SimulationActivity,
+    SimulationActivityState,
+} from './simulation-activity';
+
+export class TransferPatientToHospitalActivityState
+    implements SimulationActivityState
+{
+    @IsValue('transferPatientToHospitalActivity' as const)
+    public readonly type = 'transferPatientToHospitalActivity';
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly id: UUID;
+
+    @IsUUIDSet()
+    public readonly patientIds: UUIDSet;
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly vehicleId: UUID;
+
+    @IsUUID(4, uuidValidationOptions)
+    public readonly transferManagementRegionId: UUID;
+
+    /**
+     * @deprecated Use {@link create} instead
+     */
+    constructor(
+        id: UUID,
+        patientIds: UUIDSet,
+        vehicleId: UUID,
+        transferManagementRegionId: UUID
+    ) {
+        this.id = id;
+        this.patientIds = patientIds;
+        this.vehicleId = vehicleId;
+        this.transferManagementRegionId = transferManagementRegionId;
+    }
+
+    static readonly create = getCreate(this);
+}
+
+export const transferPatientToHospitalActivity: SimulationActivity<TransferPatientToHospitalActivityState> =
+    {
+        activityState: TransferPatientToHospitalActivityState,
+        tick(
+            draftState,
+            simulatedRegion,
+            activityState,
+            tickInterval,
+            terminate
+        ) {
+            const vehicle = getElement(
+                draftState,
+                'vehicle',
+                activityState.vehicleId
+            );
+            const patients = Object.keys(activityState.patientIds).map(
+                (patientId) => getElement(draftState, 'patient', patientId)
+            );
+            const transferManagementRegion = getElement(
+                draftState,
+                'simulatedRegion',
+                activityState.transferManagementRegionId
+            );
+
+            vehicle.occupation = cloneDeepMutable(
+                IntermediateOccupation.create(
+                    draftState.currentTime + tickInterval
+                )
+            );
+
+            sendSimulationEvent(
+                simulatedRegion,
+                TransferPatientsInSpecificVehicleRequestEvent.create(
+                    activityState.patientIds,
+                    activityState.vehicleId,
+                    'hospital',
+                    catchAllHospitalId
+                )
+            );
+
+            patients.forEach((patient) => {
+                sendSimulationEvent(
+                    transferManagementRegion,
+                    PatientTransferToHospitalSuccessfulEvent.create(
+                        Patient.getVisibleStatus(
+                            patient,
+                            draftState.configuration.pretriageEnabled,
+                            draftState.configuration.bluePatientsEnabled
+                        ),
+                        simulatedRegion.id
+                    )
```

Comment:
We have decided to keep it as it is, since

- it would require a lot of changes to pass the information that a report should be sent trough all the layers of the transfer framework
- in a well-configured scenario (i.e., all required behaviors exist on the simulated region), the transfer should not fail
- it is realistic that the success of some operation might be reported to early